contracts/src/CrossChainApplications/ERC20Bridge/ERC20Bridge.sol

```diff
@@ -570,18 +581,16 @@ contract ERC20Bridge is IERC20Bridge, ITeleporterReceiver, ReentrancyGuard {
         uint256 feeAmount
     ) private {
         // Do not allow nested bridging of wrapped tokens.
-        require(
-            !wrappedTokenContracts[nativeContractAddress],
-            "Cannot bridge wrapped token."
-        );
+        if (wrappedTokenContracts[nativeContractAddress]) {
+            revert WrappedTokenDoesNotExist(nativeContractAddress);
```

Comment:
Seems like an odd error for this case. Shouldn't this revert using something like "CannotBridgeWrappedToken" instead?